.travis.yml

```diff
@@ -7,7 +7,7 @@ python:
 env:
  - DJANGO_VERSION="Django>=1.8,<1.9"
  - DJANGO_VERSION="Django>=1.9,<1.10"
- - DJANGO_VERSION="Django==1.10b1"
+ - DJANGO_VERSION="Django>=1.10"
```

Comment:
Thought about it but figured we'd not really need it but I'll make the change since explicit is always better